.github/workflows/ci.yml

```diff
@@ -34,7 +33,7 @@ jobs:
       - name: Setup dart
         uses: dart-lang/setup-dart@v1
       - name: Lint
-        run: dart format --output=none --set-exit-if-changed packages/dart
+        run: dart format --output=none --set-exit-if-changed /
```

Comment:
dart format --output=none --set-exit-if-changed `./`

Reply:
It was done